src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,127 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Entry first;
+    private Entry last;
+    private int size;
+
+    public MyLinkedList() {
+        this.first = new Entry(null, last, last);
+        this.last = new Entry(null, first, first);
+    }
+
+    private static class Entry<T> {
+        T element;
+        Entry<T> next;
+        Entry<T> prev;
+
+        Entry(T element, Entry<T> next, Entry<T> prev) {
+            this.element = element;
+            this.next = next;
+            this.prev = prev;
+        }
+    }
+
+    private Entry iterator(int index) {
+        Entry currentElement = first;
+        for (int i = 0; i <= index; i++) {
+            currentElement = currentElement.next;
+            if (i == index) {
+                return currentElement;
+            }
+        }
+        return null;
+    }
+
     @Override
     public boolean add(T value) {
-        return false;
+        Entry newEntry = new Entry(value, last, last.prev);
```

Comment:
```suggestion
        Entry<T> newEntry = new Entry<>(value, last, last.prev);
```

Reply:
make your entries parameterized  where you use them

Reply:
done